content/en/docs/concepts/overview/working-with-objects/_index.md

```diff
@@ -120,6 +120,22 @@ satisfy the StatefulSet specification.
 Different kinds of object can also have different `.status`; again, the API reference pages
 detail the structure of that `.status` field, and its content for each different type of object.
 
+### Server Side Field Validation
```

Comment:
```suggestion
## Server side field validation
```

(**or** make “Required fields” a second-level heading, and leave this one as is)